api/plan.js

```diff
@@ -22,6 +22,7 @@ const {
   deletePlan,
```

Comment:
This version of api/plan.js is quite a bit behind master.
The function you are modifying doesn't exist anymore, and there are no references to req.auth.userRole in the current version of api/plan.js so there isn't a need to require("../models/user"); in this file either.

I would suggest simply using the current version of api/plan.js from master.

Reply:
Done.